api-report/container-definitions.api.md

```diff
@@ -227,6 +227,7 @@ export interface IContainerEvents extends IEvent {
     (event: "op", listener: (message: ISequencedDocumentMessage) => void): any;
     (event: "dirty", listener: (dirty: boolean) => void): any;
     (event: "saved", listener: (dirty: boolean) => void): any;
+    (event: "signal", listener: (message: IInboundSignalMessage, local: boolean) => void): any;
```

Comment:
i don't think we should expose this on IContainer. It doesn't match the exiting layering/separation of concerns for the encapsulated model

Reply:
root data object would be a better place to expose signals for azure client to consume